dev/com.ibm.ws.ejbcontainer.timer.persistent/src/com/ibm/ws/ejbcontainer/timer/persistent/osgi/internal/EJBPersistentTimerRuntimeImpl.java

```diff
@@ -118,6 +131,14 @@ protected void modified(ComponentContext cc, Map<String, Object> properties) {
 
     private void updateConfiguration(Map<String, Object> properties) {
 
+        // Read the missedPersistentTimerAction configuration; do not set a default as
+        // that will vary based on whether the PersistentExecutor has failover enabled
+        String missedTimerActionProperty = (String) properties.get(MISSED_TIMER_ACTION);
+        missedTimerAction = (missedTimerActionProperty != null) ? MissedTimerAction.valueOf(missedTimerActionProperty) : null;
```

Comment:
`MissedTimerAction.valueOf(missedTimerActionProperty.toUpperCase())`

^ depending on if kernel lets metatype values be case insensitive.

Reply:
Added 2 new test variations to check for mixed case value (kernel changes to match case in metatype) and bad value (kernel logs warning and provides no value).